srcpkgs/wire-desktop/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'wire-desktop'
+pkgname=wire-desktop
+version=3.6.2885
+revision=1
+hostmakedepends="nodejs yarn git"
+short_desc="Wire for desktop"
+maintainer="Young Jin Park <youngjinpark20@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://wire.com"
+distfiles="https://github.com/wireapp/wire-desktop/archive/linux/${version}.tar.gz"
+checksum=524db450d41a2dc677b8450cdf4e13dcc753280b2f6565ed5e23e6f59c14bed4
+wrksrc="${pkgname}-linux-${version}"
+
+pre_build() {
+	sed -i '/node-addressbook/d' "electron/package.json"
```

Comment:
please change `sed` to `vsed` so that it will warn you in future version upgrades if this function stops being useful

Reply:
Also no need to quote the filename